### ISSUE TITLE
Permanently disable the prompt asking "Preloaded files has reached the upper limit..."

### ISSUE THREAD

**Issue by reportaman** [xiyaowong/coc-sumneko-lua#18]
How to permanently disable the prompt that pops up when opening a .lua file:

```
Preloaded files has reached the upper limit (0), you need to manually open the files that need to be loaded.
(1)[object Object]:
```

<img width="1019" alt="Screen Shot 2022-02-12 at 7 35 24 PM" src="https://user-images.githubusercontent.com/71736629/153737396-a85dab6b-b614-4522-84b1-6765a5041503.png">

**Comment by reportaman** [comment]
\+ @chemzqm

**Comment by reportaman** [comment]
Doesn't work... it now gives two prompts at launch time if file is in same directory while launching nvim ``nvim filename.lua``. Or gives just one prompt shown below if nvim is launched from home directory: ``nvim ~/path/to/file/filename.lua``.

<img width="1029" alt="Screen Shot 2022-02-12 at 8 37 44 PM" src="https://user-images.githubusercontent.com/71736629/153738877-de070a61-cc61-4067-b6a0-fe360b5efb73.png">

**Comment by reportaman** [comment]
Yes, what do you want to me to set it to? And why does this prompt have to do anything with size of files??

**Comment by chemzqm** [comment]
My suggestion is use `set cmdheight=2` or use `"coc.preferences.enableMessageDialog": true` to avoid `Press ENTER` (mostly)

**Comment by reportaman** [comment]
Where can a feature request be put to disable the prompt? If I have disabled preload of files entirely, IMO I shouldn't be getting a prompt.

**Comment by chemzqm** [comment]
> Where can a feature request be put to disable the prompt? If I have disabled preload of files entirely, IMO I shouldn't be getting a message and prompt.

You can use `"coc.preferences.messageLevel"` to set message level, but disable prompt will not exists with coc.nvim, you can request the language server maintainer to provide such option.

**Comment by xiyaowong** [comment]
![image](https://user-images.githubusercontent.com/47070852/153760491-ceb77232-2d15-4a09-b787-df4c8c8ea9ca.png)

Set larger values for these options @reportaman
